lib/scopes/form526_submission_state.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+module Scopes
+  module Form526SubmissionState
+    extend ActiveSupport::Concern
+
+    # DOCUMENTATION:
+    # https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/engineering_research/526_scopes.md
+    included do
```

Comment:
To bypass the linting alert, you could precede L9 with `# rubocop:disable Metrics/BlockLength` and follow the end of the block with `# rubocop:enable Metrics/BlockLength`.

Reply:
advised by platform support not to do this : /

Reply:
Eric's explanation makes sense. Sorry for steering you wrong.